app/constant.ts

```diff
@@ -91,7 +91,7 @@ Latex block: $$e=mc^2$$
 export const SUMMARIZE_MODEL = "gpt-3.5-turbo";
 
 export const KnowledgeCutOffDate: Record<string, string> = {
-  default: "2021-09",
+  default: "2023-04",
```

Comment:
this `default: "2023-04",` must made sperate for azure as example, because it will affected for openai and become bad hallucination.